### PR TITLE
Added YAML Build and ARM template for new service

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        "resourceEnvironmentName": {
+            "type": "string"
+        },
+        "loggingRedisConnectionString": {
+            "type": "string"
+        },
+        "loggingRedisKey": {
+            "type": "string"
+        },
+        "CustomHostName": {
```
Comment (chelnak) [inline azure/template.json]: Parameters should be cammelCase 🐫

File: azure/template.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            "type": "string"
+        },
+        "KeyVaultCertificateName": {
```
Comment (chelnak) [inline azure/template.json]: Parameters should be cammelCase 🐫

File: azure/template.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            "type": "Microsoft.Resources/deployments",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'), 'app-service.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "appServiceName": {
+                        "value": "[variables('AppServiceName')]"
```
Comment (chelnak) [inline azure/template.json]: Variable references should be cammelCase 🐫

File: azure/template.json [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                                "value": "[parameters('loggingRedisKey')]"
+                            },
+                            {
+                                "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                                "value": "[reference('api-app-insights').outputs.InstrumentationKey.value]"
+                            }
+                        ]
+                    },
+                    "customHostName": {
+                        "value": "[parameters('CustomHostName')]"
```
Comment (chelnak) [inline azure/template.json]: Variable references should be cammelCase 🐫

File: azure/template.json [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+            "type": "Microsoft.Resources/deployments",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'), 'application-insights.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "appInsightsName": {
+                        "value": "[variables('AppServiceName')]"
```
Comment (chelnak) [inline azure/template.json]: Variable references should be cammelCase 🐫

File: azure/template.json [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+                    },
+                    "attachedService": {
+                        "value": "[variables('AppServiceName')]"
```
Comment (chelnak) [inline azure/template.json]: Variable references should be cammelCase 🐫

File: azure/template.json (hunk 1 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 3 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 5 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 6 above)
Comment (mwevill) [reply azure/template.json]: Fixed

**Pull request merged**